include path and mod name in sq api

uniboi opened the pull request.

uniboi committed:
include path and mod name in sq api

Alystrasz commented:
@uniboi still wanna work on this?
Last time I checked, scrollbar was completely messed up.

uniboi commented:
It's still on my bucket list

catornot committed:
Support loading plugins from subdirs (#460)

Support loading plugins from subdirs to allow for better organising plugins and easier mod-manager installation

uniboi committed:
Merge branch 'main' of https://github.com/R2Northstar/NorthstarLauncher into bad-mod-info

uniboi committed:
use structs for the script mod menu

uniboi committed:
Merge branch 'feat/bad-mod-formatting-warnings' of https://github.com/Alystrasz/NorthstarLauncher into bad-mod-info

uniboi committed:
Merge branch 'feat/bad-mod-formatting-warnings' into bad-mod-info

uniboi committed:
Merge branch 'bad-mod-info' of github.com:uniboi/NorthstarLauncher into bad-mod-info

uniboi committed:
remove duplicate NSGetInvalidMods definition

uniboi committed:
formatting

Alystrasz reviewed (approved):
Code looks good, confirmed working on Ubuntu 23.04.

![image](https://github.com/Alystrasz/NorthstarLauncher/assets/11993538/99f8f236-38bc-4d9e-bdaa-937710b09cae)
